ATSQuadroStrategyBase/NinjaTrader 8/bin/Custom/Strategies/AlgoSystemBase.cs

```diff
@@ -2607,7 +2607,7 @@ public virtual StrategyTradeWorkFlowState ProcessWorkFlow(StrategyTradeWorkFlowS
                 case StrategyTradeWorkFlowState.GoShortCancelWorkingOrdersConfirmed:
 
                     TradeWorkFlowOnMarketDataDisable();
-                    if (IsHistoricalTradeOrPlayBack || IsStrategyUnSafeMode)
+                    if (IsStrategyUnSafeMode)

```

Comment:
This is required - IsHistoricalTradeOrPlayBack || IsStrategyUnSafeMode use the same paths a lot of the time-